src/components/ScenarioParameters/components/ScenarioParametersTabs/ScenarioParametersTabsWrapper.js

```diff
@@ -43,25 +33,15 @@ const ScenarioParametersTabsWrapper = ({
     if ('name' in tabFactory || (tabFactory?.WrappedComponent && 'name' in tabFactory.WrappedComponent)) {
```

Comment:
the same question: can we remove the condition checking `name` ? :slightly_smiling_face: